opencodelists/settings.py

```diff
@@ -175,6 +179,15 @@
 }
 # fmt: on
 
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+if SENTRY_DSN:
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration()],
+        # If you wish to associate users to errors (assuming you are using
+        # django.contrib.auth) you may enable sending PII data.
```

Comment:
I'm not quite sure who this comment's for.

Reply:
Ah sorry, the whole thing was copied verbatim from Sentry's docs.